windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -0,0 +1,65 @@
+
+[id="wmco-3-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 3.0.0
+
+This release of the WMCO provides bug fixes and enhancements for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 3.0.0 were released in link:https://errata.devel.redhat.com/advisory/76465[RHSA-2021:76465-05].
+
+include::modules/wmco-prerequisites.adoc[leveloffset=+1]
+
+
+include::modules/wmco-prerequisites.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+Running Windows container workloads is not supported for clusters in a restricted network or disconnected environment.
+====
+
+Version 3.x of the WMCO is only compatible with {product-title} 4.8.
+
+[id="wmco-3-0-0-improvements"]
+=== New features and improvements
+
+This release adds the following new features and improvements.
+
+// Addresses Github #32620 - https://github.com/openshift/openshift-docs/pull/32620
+==== Clarified limits on custom VXLAN port selection
+
+Users may not select a custom VXLAN port when using the latest version of Windows server.
+
+[id="wmco-3-0-0-bug-fixes"]
+=== Bug fixes
+
+* Previously, The load balancer service would become unstable when the backing deployment had multiple pods scheduled on different Windows nodes. This issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905950[**BZ#1905950**])
+
+* Previously, WMCO added the public key annotation, `windowsmachineconfig.openshift.io/pub-key-hash`, to Linux nodes. Now, WMCO no longer adds an annotation to Linux nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930791[**BZ#1930791**])
+
+* Previously, the kube-proxy service would crash upon the creation of a load balancer service when the backing development had multiple pods scheduled on different Windows nodes. This issue has now been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[**BZ#1939968**])
+
+* Previously, the `windows_exporter` component reported various metrics as `windows_*`. This error caused some of the node-level metrics that provide insight into the nodes through telemetry services to go unreported. Now, the componets export correctly showing all of the expected metrics. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1948037[**BZ#1948037**])
+
+=== RHSA-2021:76465-04 - Windows Container support for {product-title} security update
+
+As part of the previously noted bug fix (link:https://bugzilla.redhat.com/show_bug.cgi?id=1946538[**BZ#1946538**]), an update for Windows kube-proxy is now available for Red Hat Windows Machine Config Operator 2.0.1. Details of the update are documented in the link:https://access.redhat.com/errata/RHSA-2021:2130[RHSA-2021:2130] advisory.
+
+[id="wmco-3-0-0-known-issues"]
+=== Known issues
```

Comment:
The Known issues section is empty. Should a statement be added that there are no issues or should the heading be removed (depending on current pattern)?